### PR TITLE
Push labels to PDS

### PR DESCRIPTION
**Pull request by devinivy**
Allows the appview to push labels to the PDS using an unspecced route, as a temporary measure.  Also adds an http retry to the hive labeler.

### CODE REVIEW

File: lexicons/app/bsky/unspecced/applyLabels.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          "required": ["subject"],
+          "properties": {
+            "subject": {
+              "type": "union",
+              "refs": [
+                "com.atproto.admin.defs#repoRef",
+                "com.atproto.repo.strongRef"
+              ]
+            },
+            "createLabelVals": {
```
Comment (dholms) [inline lexicons/app/bsky/unspecced/applyLabels.json:21]: i kinda want this to just take already formatted labels 🤔 

so all the pds has to do is dumb shuffling of them off to the database, no formatting or anything & nowhere that the labels could get out of sync

File: packages/bsky/src/labeler/hive.ts [hunk 2]
```diff
@@ -51,13 +52,16 @@ export const makeHiveReq = async (
...
   form.append('media', blob)
-  const res = await axios.post(HIVE_ENDPOINT, form, {
-    headers: {
-      'Content-Type': 'multipart/form-data',
-      authorization: `token ${hiveApiKey}`,
-      accept: 'application/json',
-    },
-  })
+
+  const res = await retryHttp(() =>
```
Comment (dholms) [inline packages/bsky/src/labeler/hive.ts:56]: this does max of 3 right? seems good 👍

**Review by dholms — APPROVED** [review]
seems solid 👍 

as i mentioned, i'm inclined to do this with full label objects instead of `create/deleteLabelVals`. Just seems simpler & less prone to small bugs. But also not married to it & think this is totally fine

File: lexicons/app/bsky/unspecced/applyLabels.json (hunk 1 above)
Comment (devinivy) [reply lexicons/app/bsky/unspecced/applyLabels.json:21]: Yeah, I like that more—on it 👍

File: packages/bsky/src/labeler/hive.ts (hunk 2 above)
Comment (devinivy) [reply packages/bsky/src/labeler/hive.ts:56]: Yup!

**Review by dholms — APPROVED** [review]
:shipit:

**Pull request merged**